doc/api/Loading_a_Content.md

```diff
@@ -306,8 +306,20 @@ This object can take the following properties (all are optional):
 
       Defaults to `4`.
 
-    - `timeout` (`number|undefined`): Timeout, in milliseconds, after which
-      segment requests are aborted and, depending on other options, retried.
+    - `timeout` (`number|undefined`): Specifies the maximum time, in milliseconds, 
+      that the client can wait for downloading all the data of the request response.
+      If all data are not downloaded within the specified timeout, segment request are
+      aborted and, depending on other options, retried.
+
+      To set to `-1` for no timeout.
+
+      `undefined` (the default) will lead to a default, large, timeout being
+      used.
+
+    - `connectionTimeout` (`number|undefined`): Specifies the maximum time, in milliseconds, 
+      that the client can wait for receiving the responses headers and status code.
+      If they are not received within the specified timeout, segment requests are aborted and,
+      depending on other options, retried.
```

Comment:
Nice. Maybe it would also make sense to add a short sentence to explain the difference with just `timeout`

Reply:
Done :)